plugin/src/main/resources/reference.conf

```diff
@@ -5,16 +5,9 @@ play {
 
   cache {
     # The name of the resource that should be used to configure the cache
-    # jcacheConfig
-
-    # The caches to bind
-    bindCaches = []
```

Comment:
Why do you remove that reference? Because you assume users know the valid props in the Cache module itself?